packages/plugins/src/hardhat/tasks.ts

```diff
@@ -90,6 +90,9 @@ export const TASK_CHUGSPLASH_MONITOR = 'chugsplash-monitor'
 export const TASK_CHUGSPLASH_CANCEL = 'chugsplash-cancel'
 export const TASK_CHUGSPLASH_WITHDRAW = 'chugsplash-withdraw'
 export const TASK_CHUGSPLASH_LIST_PROJECTS = 'chugsplash-list-projects'
+export const TASK_CHUGSPLASH_TRANSFER_OWNERSHIP =
+  'chugsplash-transfer-ownership'
+export const TASK_CHUGSPLASH_CLAIM_OWNERSHIP = 'chugsplash-claim-ownership'
```

Comment:
What do you think about changing this task name to `chugsplash-claim-proxies`? I feel like "claim ownership" could either mean claiming ownership of the ChugSplashManager or claiming ownership of the proxies.